### PR TITLE
Added midi save feature

### PR DESCRIPTION
**Pull request by JamesTheHacker**

### CODE REVIEW

File: src/writer.js [hunk 1]
```diff
@@ -55,11 +55,24 @@ class Writer {
...
+		 */
     stdout() {
     	return process.stdout.write(new Buffer(this.buildFile()));
     }
+
+		/**
+		 * Save to MIDI file
+		 *
+		 */
+		 saveMIDI(filename) {
```
Comment (grimmdude) [inline src/writer.js]: Hey thanks a lot for this PR.  You mind adjusting the indentation here to use tabs?

Comment (grimmdude) [inline src/writer.js]: That is, 4 space length tabs.

**Comment by JamesTheHacker** [comment]
Will fix this in the morning. I'm just about to head home :)

**Comment by grimmdude** [comment]
Thanks for the PR!

**Pull request merged**